### ISSUE TITLE
@graphql-codegen/cli: A 'this' type is available only in a non-static member of a class or interface.

### ISSUE THREAD

**Issue by shinyaohira** [dotansimha/graphql-code-generator#3697]
**Describe the bug**
I cannot compile my code with `generate` function of @graphql-codegen/cli.

```
$ tsc -p .
node_modules/@graphql-codegen/cli/config.d.ts:12:70 - error TS2526: A 'this' type is available only in a non-static member of a class or interface.

12 export declare function setCommandOptions(commandInstance: Command): this;
```

**To Reproduce**

```typescript
import { generate } from "@graphql-codegen/cli";

...
  return generate({
    schema: "http://xxx/",
    generates: {
      [productionSchemaPath]: {
        plugins: ["introspection"]
      }
    }
  });
...
```

**Expected behavior**
No compile error.

**Environment:**

- OS: macOS Catalina
- `@graphql-codegen/...`: v1.13.1
- NodeJS: v12.11.1

**Additional context**

**Comment by robbyemmert** [comment]
I'm getting this issue as well. Any ETA when it will be available?

**Comment by kamilkisiela** [comment]
You can enable `compilerOptions.skipLibCheck` in tsconfig.json to temporary fix it, until we get a release.

**Comment by dotansimha** [comment]
Fixed in v1.13.2
